File: app/src/main/java/com/penseapp/acaocontabilidade/chat/interactor/ChatsWriterInteractor.java
package com.penseapp.acaocontabilidade.chat.interactor;

/**
 * Created by Cleusa on 27/11/2016.
 */

public interface ChatsWriterInteractor {
    void createChatIfNeeded(String senderId, String senderName, String recipientId, String recipientName);
}

File: app/src/main/java/com/penseapp/acaocontabilidade/chat/chats/interactor/ChatsWriterInteractor.kt
package com.penseapp.acaocontabilidade.chat.chats.interactor

/**
 * Created by Cleusa on 27/11/2016.
 */

interface ChatsWriterInteractor {

    fun createChatIfNeeded(senderId: String,
                           senderName: String,
                           senderCompany: String,
                           recipientId: String,
                           recipientName: String,
                           recipientCompany: String)
}
